backend/src/v2/component/launcher_v2.go

```diff
@@ -1,3 +1,16 @@
+// Copyright 2021-2023 The Kubeflow Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
NIT: should we replace the `\tab` with spaces to be consistent with other license headers?

Reply:
I will replace the tabs with 6 spaces, as is consistent with other copyright declarations.